client/post-editor/media-modal/image-editor/index.jsx

```diff
@@ -73,9 +77,27 @@ const MediaModalImageEditor = React.createClass( {
 	onImageExtracted( blob ) {
 		const mimeType = MediaUtils.getMimeType( this.props.fileName );
 
+		// check if a title is already post-fixed with '(edited copy)'
+		const editedCopyText = this.props.translate(
+			'%(title)s (edited copy)', {
+				args: {
+					title: ''
+				}
+			} );
+		let title = this.props.title;
+		if ( title.indexOf( editedCopyText ) === -1 ) {
```

Comment:
It's a bit hacky, but the key here is to ensure that we use the exact same translation string. As long as this is pre or post fixed in each i18n translation, the matching should work. We will hit some false positives should the user ever have a title like this, but in that case having an additional `(edited copy)` would look odd there anyway.

Any fancier cases, like `(edited copy 3)` will require additional state being stored on the media object, which we can revisit if we'd really like